tools/test/travis-ci/scancode-evaluate.py

```diff
@@ -75,6 +76,18 @@ def has_spdx_text_in_analysed_file(scanned_file_content):
     return bool(re.findall("SPDX-License-Identifier:?", scanned_file_content))
 
 
+def has_binary_license(scancode_output_data_file):
+    """Returns true if the file analysed by ScanCode contains a Permissive Binary License."""
+    file_path = os.path.abspath(scancode_output_data_file['path'])
+    try:
+        with open(file_path, 'r') as read_file:
+            scanned_file_content = read_file.read()
+        return bool(re.findall("Permissive Binary License", scanned_file_content))
+    except UnicodeDecodeError:
+        userlog.warning("Unable to look for PBL text in `{}`:".format(file_path))
```

Comment:
I think the recommendation for logging functions is 
`Use % formatting in logging functions and pass the % parameters as arguments`